.github/workflows/docs-test.yaml

```
@@ -34,14 +34,17 @@ jobs:
         uses: actions/cache@v3
         with:
           path: .venv
-          key: venv-docs-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
+          key: venv-docs-dummy2-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
```

```suggestion
          key: venv-docs-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
```
Do we still need this change?

---

nope :)